index.html

```diff
@@ -269,111 +269,111 @@ <h2 class="font-work-sans font-semibold pl-8 text-2xl text-center">
                         class="w-32 flex flex-col justify-center items-center rounded-xl font-work-sans p-2 cursor-pointer"
                     >
                         <img
-                            src="./resources/assets/jsloversLogo.png"
+                            src="./resources/assets/microsoft_logo.png"
                             alt="jslovers homepage"
                             width="100"
                         />
 
-                        <h4 class="mt-2 underline">Microsoft</h4>
+                        <!-- <h4 class="mt-2 underline">Microsoft</h4> -->
                     </div>
                     <div
                         class="w-32 flex flex-col justify-center items-center rounded-xl font-work-sans p-2 cursor-pointer"
                     >
                         <img
-                            src="./resources/assets/jsloversLogo.png"
+                            src="./resources/assets/expedia_logo.svg"
                             alt="jslovers homepage"
                             width="100"
                         />
 
-                        <h4 class="mt-2 underline">Expedia</h4>
+                        <!-- <h4 class="mt-2 underline">Expedia</h4> -->
                     </div>
                     <div
                         class="w-32 flex flex-col justify-center items-center rounded-xl font-work-sans p-2 cursor-pointer"
                     >
                         <img
-                            src="./resources/assets/jsloversLogo.png"
+                            src="./resources/assets/google_logo.png"
                             alt="jslovers homepage"
                             width="100"
```

Comment:
Can we please put a class and there we assign the width rather than having inline CSS?

Reply:
done